LICENSES/GPL-2.0-or-later.txt

```diff
@@ -0,0 +1,103 @@
+GNU GENERAL PUBLIC LICENSE
```

Comment:
Why is this  here? It isn't mentioned in .reuse/dep5.

Reply:
It is one of the dual licenses on lib/zstddeclib.c.

Reply:
dep5 is only for files that don't have copyright/license information imbedded in the file itself. For this repo that is binaries/images, or files that come from elsewhere I didn't feel comfortable modifying just for that purpose.